### DISCUSSION TITLE
Are these "fake visitors" bots ?

### DISCUSSION THREAD

**Discussion by clairem44** [plausible/analytics#2739] [Anything Else]
Hi,

I have recently migrated a client of mine to Plausible from GA.
Previously we had a cookie consent banner so the visits were counted in GA only when people accepted cookie (normal).

Now with Plausible we don't need this cookie banner anymore and we have deleted it.
So : since Plausible has been implemented, the traffic figures have risen up : normal, because everyone is counted (non only those who accept cookies).

BUT : I think there is something else.

When I compare "old" analytics (with GA) and "new" analytics" (with Plausible), I see that in Plausible, Ithere is a lot of new traffic from the US that we did not have before (my client is french).

When I look further on this visitors from the US here is what I see : 
- they mostly view the homepage only
- they are mostly Direct traffic
- their visit duration is very low and the bounce rate very high
- they are mostly located in Boardman, Oregon and Ashburn, Virginia
- they mostly use Chrome (96%)

So, I guess they are fake visitors like bots.
Am I right ? Is there a way to exclude these fake visitors ?

**Comment by metmarkosaric** [comment]
thanks for reaching out @clairem44! 

it's really difficult to say whether this is bot traffic as single page visits with low visit duration and high bounce rate are also a sign of low quality traffic. we don't have a way for you to manually exclude specific segments of your traffic unfortunately. 

we do already monitor for bots and other non-human traffic by default and we exclude all that traffic. but as always, it's really difficult task to detect it all so some bots may slip through (kind of like they do in GA and in other analytics tools). you can read a bit more about how our bot detection works here: https://plausible.io/docs/dashboard-faq#does-plausible-exclude-known-bots-and-spam-traffic. 

hope this helps, thanks!

**Comment by clairem44** [comment]
Thanks for your answer @metmarkosaric !
It really looks like fake traffic according to the location.
I've looked deeper and found lots of people who report the same fake traffic from Boardman, Oregon and Ashburn, Virginia.
Both comes from AWS bots.
You can find some sources here : https://kwsmdigital.com/why-do-i-have-so-much-website-traffic-from-ashburn-2/
And here : https://www.kesslerfreedman.com/2018/10/boardman-oregon-and-your-analytics-reports/
It's been well documented, so maybe it could be something to look at, if you want to exclude these bots from Plausible analytics stats.
Thanks !
